src/js/popper.js

```diff
@@ -166,6 +165,18 @@ export function reflow(popper) {
   void popper.offsetHeight
 }
 
+/**
+ * Adds/removes theme from tooltip's classList
+ * @param {HTMLDivElement} tooltip
+ * @param {String} theme
+ * @param {String} action
+ */
+export function theme(tooltip, theme, action) {
```

Comment:
Honestly, it was hard for me to decipher this function purpose without looking at the declaration here.
Maybe I've been using data-last approach too much recently but doesn't this `theme(tooltip, props.theme, 'add')` look kinda weird to you? :D

Reply:
Yeah I forgot about this and wanted to fix it as soon as I made it, it has `theme` in scope 3 different times lol